docs/config/config.xml

```diff
@@ -26,6 +26,11 @@
     </database>
 
     <CouchDB>
+        <dbName>%COUCH_DATABASE%</dbName>
+        <hostname>%COUCH_HOSTNAME%</hostname>
+        <port>%COUCH_PORT%</port>
+        <admin>%COUCH_USERNAME%</admin>
+        <adminpass>%COUCH_PASSWORD%</adminpass>
```

Comment:
I just checked and there's nothing in the default schema config table, so it's fine.. moving it should be a different PR, it's not high priority.